db/projects/neo/project.yaml

```diff
@@ -0,0 +1,170 @@
+name: NEO
+web:
+  - 'https://neo.org/'
+start_date: '2014-02-01'
+description: 'NEO is a blockchain, originally known as AntShares, sometimes called as chinese Ethereum. It is a first open source platform for smart contracts started in China.'
+ledgers:
+  - id: neo
+    name: NEO
+    type: blockchain
+    networks:
+      - id: main
+        type: main
+        name: Main Net
+        algorithm: dBFT
+        proof_type: PoS
+        tools:
+          - type: explorer
+            url: 'https://neotracker.io'
+          - type: explorer
+            url: 'https://neoscan.io/'
+          - type: explorer
+            url: 'http://monitor.cityofzion.io/'
+          - type: explorer
+            url: 'https://scan.nel.group/#mainnet'
+          - type: explorer
+            url: 'http://state.otcgo.cn/index.html?network=mainnet&idx=0'
+          - type: explorer
+            url: 'https://blolys.com/#/mainnet'
+      - id: test
+        type: test
+        name: TestNet
+        tools:
+          - type: explorer
+            url: 'http://monitor.cityofzion.io/'
+          - type: explorer
+            url: 'https://scan.nel.group/#testnet'
+          - type: explorer
+            url: 'https://neoscan-testnet.io/'
+          - type: explorer
+            url: 'http://state.otcgo.cn/?network=testne&idx=0'
+assets:
+  - id: neo
+    description: 'A token represents the ownership of the NEO blockchain.'
+    name: NEO
+    type: native
+    ledger: neo
+    symbol: NEO
+    max_supply: 100000000
+    ico_properties:
+      start_date: '2017-08-08'
+      end_date: '2017-09-09'
+      total_cap: 100000000
+      ico_price: 0.032 USD
+      raised: 6199.53 BTC
+      country: China
+    first_distribution:
+      - title: 'public participants'
+        amount: 20000000
+        percent: 20
+    webids:
+      coinmarketcap: neo
+      cryptocompare: neo
+      cryptocompare/id: '27368'
+  - id: gas
+    description: 'A cryptocurrency which is distributed by holding NEO, used to fuel transactions on the NEO blockchain.'
+    name: Gas
```

Comment:
From [NEO whitepaper](http://docs.neo.org/en-us/whitepaper.html):
> NEO has two native tokens, NEO (abbreviated symbol NEO) and NeoGas (abbreviated symbol GAS).

So name should be "NeoGas" (you can put "Gas" to `alt_names`)

Reply:
I do agree. Even though the name NeoGas is not used anywhere, it is still correct name for the asset.

Changed to:
name: NeoGas
alt_names:
  - Gas